CITATION.cff

```diff
@@ -67,6 +67,22 @@ authors:
   family-names: Garg
   name-particle: Bala
   affiliation: Stanford University
+- given-names: Andreas
+  family-names: Stefl
+  affiliation: CERN / TU Wien
+- given-names: Luis Falda
+  family-names: Coelho
+  affiliation: CERN / University of Coimbra
+- given-names: Alexander J.
+  family-names: Pfleger
+  affiliation: CERN / University of Graz
```

Comment:
```suggestion
  affiliation: CERN / University of Graz
  orcid: https://orcid.org/0000-0001-5524-7738
```

Reply:
I've added your ORCID!

Reply:
@stephenswat since you are adding the ORCID, this is mine: https://orcid.org/0000-0002-2298-3605

Reply:
Added. :smile: